numpy/core/src/umath/_umath_tests.c.src

```diff
@@ -372,10 +391,12 @@ static struct PyModuleDef moduledef = {
 #endif
 
 #if defined(NPY_PY3K)
-#define RETVAL m
+#define RETVAL_SUCCESS m
+#define RETVAL_FAILURE NULL
```

Comment:
In `multiarraymodule.c` we use
```
#if defined(NPY_PY3K)
#define RETVAL(x) x
PyMODINIT_FUNC PyInit_multiarray(void) {
#else
#define RETVAL(x)
PyMODINIT_FUNC initmultiarray(void) {
#endif
```

would be nice to be consistent here

Reply:
Wasn't aware of that - like it much better, so will change.